### PR TITLE
Color palette for GIMP and Inkscape

### PR DESCRIPTION
**Pull request by vanillajonathan**
Color palette file that can be imported in software such as GIMP, Inkscape, etc.

### CODE REVIEW

**Review by bwinton — CHANGES_REQUESTED** [review]
So, we may change the colors in the future, which means that this file should _really_ be auto-generated like the CSS, SASS, and JSON versions.

File: colors/Photon.gpl [hunk 1]
```diff
@@ -0,0 +1,56 @@
+GIMP Palette
+Name: Firefox/Photon
+
+255 26 217 #ff1ad9
```
Comment (bwinton) [inline colors/Photon.gpl]: We should name these as per the names in the json file (i.e. `blue-50`). I don't mind if the hex color is in a comment after that, though.

File: colors/Photon.gpl [hunk 2]
```diff
@@ -0,0 +1,56 @@
+GIMP Palette
+Name: Firefox/Photon
```
Comment (bwinton) [inline colors/Photon.gpl:2]: Maybe we could put some [extra documentation](https://github.com/portnov/color-palette/wiki/Gimp-palette-format-extensions#palette-metainformation) here, like the url to this repo?

Comment (vanillajonathan) [reply colors/Photon.gpl:2]: Good idea!

File: colors/Photon.gpl (hunk 1 above)
Comment (vanillajonathan) [reply colors/Photon.gpl]: I agree.

Comment (vanillajonathan) [reply colors/Photon.gpl]: I agree!

**Comment by vanillajonathan** [comment]
I filed a new patch that generates the file.
#7

**Comment by bwinton** [comment]
(I'm going to close this, and use the auto-generated one from your other patch. Thanks!)

**Pull request closed without merging**